pages/spellbooks.html

```diff
@@ -90,26 +90,26 @@
 <a name="Arcane"></a><header class="Arc">Arcane</header><pre class="Arc">
 Cantrips for Beginners           Minor Arcana                    Major Arcana                    Manual of Mastery</pre><pre>
 Name                    Level    Name                   Level   Name                    Level   Name                    Level
-<span title=''>Zap                         1</span>    <span title=''>Detect Doors & Traps       7</span>   <span title=''>Resist Lightning           14</span>   <span title=''>See Invisible              28</span>
-<span title=''>Wizard Lock                 1</span>    <span title=''>Phlogiston                 8</span>   <span title=''>Resist Acid                15</span>   <span title=''>Resist Poison              31</span>
-<span title=''>Detect Invisibility         1</span>    <span title=''>Detect Treasure            9</span>   <span title=''>Cure Medium Wounds         16</span>   <span title=''>Teleport Level             35</span>
-<span title=''>Detect Monsters             2</span>    <span title=''>Detect Enchantment         9</span>   <span title=''>Teleport                   18</span>   <span title=''>Teleport Away              40</span>
-<span title=''>Blink                       2</span>    <span title=''>Detect Objects             9</span>   <span title=''>Identify                   20</span>   <span title=''>Recharging                 41</span>
-<span title=''>Light Area                  4</span>    <span title=''>Cure Poison               11</span>   <span title=''>Stone to Mud               22</span>   <span title=''>Detection                  42</span>
-<span title=''>Trap & Door Destruction     5</span>    <span title=''>Resist Cold               12</span>   <span title=''>Ray of Light               25</span>   <span title=''>Word of Recall             45</span>
-<span title=''>Cure Light Wounds           6</span>    <span title=''>Resist Fire               13</span>   <span title=''>Satisfy Hunger             28</span>   <span title=''>Clairvoyance               49</span>
-</pre></br>
-<a name="Trump"></a><span title="Creates granite walls in all adjacent squares."><header class="Tru">Trump</header></span><pre class="Tru">
+<span title='Bolt of lighting that does 3+((level-1)/5)d3 damage. Chance to be a beam based on class and level.'>Zap                         1</span>    <span title='Detects traps, doors, and stairs in a radius of 30.'>Detect Doors & Traps       7</span>   <span title='Gives rElec for 1d20+20 turns. Adds to gear bonuses.'>Resist Lightning           14</span>   <span title='Gives see invisibility for 1d24+24 turns.'>See Invisible              28</span>

```

Comment:
The format for Zap should be "Bolt or beam which does * electric damage."
We have a note at the top of the page describing this behaviour.